honeybot/plugins/dictionary.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+[joke.py]
+Dictionary Plugin
+
+[Author]
+Nishant, JPMorgan Chase & Co.
+
+[About]
+sends the meaning of the word
+
+[Commands]
+>>> .dictionary <<word>>
+returns a random joke
+"""
+
+from PyDictionary import PyDictionary
```

Comment:
actually you're the first plugin with external dependency. how do you think resolve that. meaning how can we inform users so that they can install it?

Reply:
we can make a requirement.txt file, which can contain all the external dependencies.
So that python automatically install the required dependencies

Reply:
@iamnishant14 from which country are you? btw made a new thread to discuss about the issue

Reply:
see issues